src/org/sosy_lab/java_smt/api/FormulaType.java

```diff
@@ -205,14 +205,61 @@ public String toSMTLIBString() {
     }
   }
 
-  public static FloatingPointType getFloatingPointType(int exponentSize, int mantissaSize) {
-    return new FloatingPointType(exponentSize, mantissaSize);
+  /**
+   * Constructs a new IEEE-754 {@link FloatingPointType} with the given exponent and mantissa sizes.
+   * The mantissa size is expected to not include the sign bit.
+   *
+   * @param exponentSize size of the exponent for the base of the floating-point
+   * @param mantissaSizeWithoutSignBit size of the mantissa (also called a coefficient or
+   *     significant), excluding the sign bit.
+   * @return the newly constructed {@link FloatingPointType}.
+   */
+  // TODO: mark as soon to be deprecated
+  public static FloatingPointType getFloatingPointType(
+      int exponentSize, int mantissaSizeWithoutSignBit) {
+    return new FloatingPointType(exponentSize, mantissaSizeWithoutSignBit);
```

Comment:
I would delegate to `getFloatingPointTypeWithoutSignBit()` here. One place less where one has to wonder whether it is correct because one cannot see the required semantics directly from the code.